content/source/docs/cloud/api/agent-tokens.md

```diff
@@ -39,6 +39,15 @@ Status  | Response                                     | Reason
 [200][] | [JSON API document][] (`type: "authentication-tokens"`) | Success
 [404][] | [JSON API error object][]                    | Agent Pool not found, or user unauthorized to perform action
 
+### Query Parameters
+
+This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
+
```

Comment:
```suggestion
This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.

```

Reply:
Thanks!  Given that the rest of the API docs are written this way going to keep as-is for consistency!